stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -1256,6 +1363,7 @@ class Stripe internal constructor(
 
     private fun createToken(
         tokenParams: TokenParams,
+        stripeAccountId: String?,
```

Comment:
can we make `stripeAccountId` come after `idempotencyKey` in the args?

Reply:
👍 on args order. imo, since this is a private method, stripeAccountId should be required so that we don't accidentally forget to supply it somewhere. Let the compiler catch anywhere that might need to be updated, instead of just letting a bunch of places silently continue using the default when they shouldn't be. Don't feel too strongly about that though.

Reply:
oh I had it in this order because I feel like we typically put required params before optional ones (except for the callback which should be last)

Reply:
that makes sense

Reply:
@mshafrir-stripe @smaskell-stripe 
This just for discussion sake, do you think it is better or recommended to put `idempotencyKey` `stripeAccount` 
Or is there any reason we could not re-use something like 
https://github.com/stripe/stripe-android/blob/d21b7fa400dacfa1a99aaf92e63542c18babaf22/stripe/src/main/java/com/stripe/android/ApiRequest.kt#L45

Reply:
Either way seems fine to me. We're creating the request options in this function anyway but we could make them a parameter.